src/views/sidebar/components/sidebar.js

```diff
@@ -14,13 +14,15 @@ export default class Sidebar extends Component {
         visibleTab: PropTypes.string,
         updateVisibility: PropTypes.func.isRequired,
         updateTabVisibility: PropTypes.func.isRequired,
-        toggleEditMode: PropTypes.func.isRequired,
+        toggleMetadataEditMode: PropTypes.func.isRequired,
+        toggleDescriptionEditMode: PropTypes.func.isRequired,
         updateMetadata: PropTypes.func.isRequired,
         saveMetadata: PropTypes.func.isRequired,
         updateTags: PropTypes.func.isRequired,
         updateDescription: PropTypes.func.isRequired,
         saveDescription: PropTypes.func.isRequired,
-        editMode: PropTypes.bool.isRequired,
+        metadataEditMode: PropTypes.bool.isRequired,
+        descriptionEditMode: PropTypes.bool.isRequired,
```

Comment:
@tjboswell While it was a gigantic pain for me to thread this through the first time, on reflection, do you think it'd be better to pass an object called `editMode` where we store a bunch of edit modes?